### DISCUSSION TITLE
A file that is not added or committed on BioDrop

### DISCUSSION THREAD

**Discussion by adiati98** [EddieHubCommunity/LinkFree#10028] [Q&A]
Hi BioDrop maintainers,

I started over and cloned the BioDrop repo to delete my `.json` file.
Right after I cloned and opened the repo on VS Code, I saw a file that is not added and committed.

Should I remove this file or add and commit it with my PR later?

![biodrop status](https://github.com/EddieHubCommunity/BioDrop/assets/45172775/ec963777-0957-464e-afa5-11942d1f3966)

**Comment by ikrishg** [comment]
It says you modified that file, could have been a formatting issue

I would simply run the following commands to reset the repo and then delete the file committing it

```bash
# reset head and deleted all commits/progress
git reset --hard HEAD~1

# bring in the changes from the remote
git pull

# create a new branch
git checkout -b data-delete-$YOUR_NAME

# deleting the file

rm -rf data/$YOUR_NAME.json

# adding and committng

git add data
git commit -m "data: delete $YOUR_NAME"

# pushing
git push --set-upstream origin data-delete-$YOUR_NAME
```

> REMEMBER TO REPLACE $YOUR_NAME with your username
> It will only work if you have cloned the forked repo and not the main EddieHub repo

**Reply by adiati98** [reply]
I just cloned it, run `git status`, and get this file right away on my terminal. I haven't even touched any files. I'm not sure why.

I'll run these command then. Thanks, @xkrishguptaa :)

**Reply by eddiejaoude** [reply]
This should be fixed now for new clones - the problem was there were 2 files with the same path+name but with different case
